spring-context/src/main/java/org/springframework/scheduling/annotation/EnableScheduling.java

```diff
@@ -180,7 +180,7 @@
  * instead of a custom <em>{@code Trigger}</em> implementation; this is because the
  * {@code task:} namespace {@code scheduled} cannot easily expose such support. This is
  * but one demonstration how the code-based approach allows for maximum configurability
- * through direct access to actual componentry.
+ * through direct access to actual component.
```

Comment:
This is not a typo: "componentry" is semantically correct here.

Although the wording could be improved, the suggested change is grammatically incorrect.